tests/pkg_fatfs/Makefile

```diff
@@ -50,4 +48,5 @@ image:
 compressed-image:
 	@./create_fat_image_file.sh $(FATFS_IMAGE_FILE_SIZE_MIB)
 
+include ../Makefile.tests_common
```

Comment:
I would also put this one before the `ifeq ($(BOARD),native)` even if the default is already set to `native`.